rqd/tests/rqcore_tests.py

```diff
@@ -567,6 +567,7 @@ def setUp(self):
 
     @mock.patch('platform.system', new=mock.Mock(return_value='Linux'))
     @mock.patch('tempfile.gettempdir')
+    @mock.patch('rqd.rqcore.pipe_to_file', new=mock.MagicMock())
```

Comment:
If we have to mock this, it means we're testing the pipe_to_file codepath instead of the default one, right?

Can we set `RQD_PREPEND_TIMESTAMP = False` in this test so we're testing the default codepath instead?